lighthouse-core/lib/traces/metrics-evts.js

```diff
@@ -0,0 +1,204 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const log = require('../../../lighthouse-core/lib/log.js');
+
+class Metrics {
+
+  constructor(traceEvents, auditResults) {
+    this._traceEvents = traceEvents;
+    this._auditResults = auditResults;
+    this._fakeEvents = [];
+  }
+
+  /**
+   * Returns simplified representation of all metrics
+   * @return {!Array<{getTs: Function, id: string, name: string}>} metrics to consider
+   */
+  static get metricsDefinitions() {
+    return [
+      {
+        name: 'Navigation Start',
+        id: 'navstart',
+        getTs: auditResults => {
+          const fmpExt = auditResults['first-meaningful-paint'].extendedInfo;
+          return fmpExt.value.timestamps.navStart;
+        }
+      },
+      {
+        name: 'First Contentful Paint',
+        id: 'ttfcp',
+        getTs: auditResults => {
+          const fmpExt = auditResults['first-meaningful-paint'].extendedInfo;
+          return fmpExt.value.timestamps.fCP;
+        }
+      },
+      {
+        name: 'First Meaningful Paint',
+        id: 'ttfmp',
+        getTs: auditResults => {
+          const fmpExt = auditResults['first-meaningful-paint'].extendedInfo;
+          return fmpExt.value.timestamps.fMP;
+        }
+      },
+      {
+        name: 'Perceptual Speed Index',
+        id: 'psi',
+        getTs: auditResults => {
+          const siExt = auditResults['speed-index-metric'].extendedInfo;
+          return siExt.value.timestamps.perceptualSpeedIndex;
+        }
+      },
+      {
+        name: 'First Visual Change',
+        id: 'fv',
+        getTs: auditResults => {
+          const siExt = auditResults['speed-index-metric'].extendedInfo;
+          return siExt.value.timestamps.firstVisualChange;
+        }
+      },
+      {
+        name: 'Visually Complete 85%',
+        id: 'vc85',
+        getTs: auditResults => {
+          const siExt = auditResults['time-to-interactive'].extendedInfo;
+          return siExt.value.timestamps.visuallyReady;
+        }
+      },
+      {
+        name: 'Visually Complete 100%',
+        id: 'vc',
```

Comment:
sounds good. 
I had chosen 'vc' origianlly because WebPageTest uses `vc` for the same thing. _OR SO I THOUGHT!_. Looking again... i can't see "vc" anywhere, so yeah i'm happy to tweak the id.